src/Microsoft.AspNetCore.Http.Abstractions/Authentication/AuthenticationManager.cs

```diff
@@ -20,18 +20,11 @@ public abstract class AuthenticationManager
 
         public abstract IEnumerable<AuthenticationDescription> GetAuthenticationSchemes();
 
-        public abstract Task AuthenticateAsync(AuthenticateContext context);
+        public abstract Task<AuthenticateInfo> GetAuthenticateInfoAsync(string authenticationScheme);
```

Comment:
FWIW, since this is the main "API", this one should probably be named `AuthenticateAsync` (as indicated by `AuthenticateInfo`'s XML documentation, BTW). The other one is just a wrapper, so it may make more sense to switch the two names.